### PR TITLE
[T5A1][T15-B2]Lam Jouq Hang Hendersen

### PR DESCRIPTION
**Pull request by takukou**

### CODE REVIEW

**Comment by takukou** [comment]
Ready for Review

**Comment by takukou** [comment]
Ready for Review

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -73,13 +76,48 @@ Use case ends.
...
 
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be able to launch application within 8 seconds.
+6. Should be able to perform add or delete function within a second.
+7. Should be user-friendly: software and functions should be easy to navigate to and usage should be intuitive.
```
Comment (K1ang) [inline doc/DeveloperGuide.md]: This point can be separated into two different requirements to make them more precise.

**Comment by K1ang** [comment]
@takukou 
Good job! Good to see the changes after our discussion in class. Keep it up!

**Pull request closed without merging**